### ISSUE TITLE
Request: snap package.

### ISSUE THREAD

**Issue by ryanpcmcquen** [keybase/client#9191]
I see some work already done here: https://github.com/keybase/client/tree/master/packaging/linux/snap

How far along is it? I'm not on one of the three distros at this point, and having a snap of Keybase would be awesome!

**Comment by funnylookinhat** [comment]
Bumping this issue - would love to see a snap.

**Comment by om26er** [comment]
@elopio Could you update on the status of the work you did, are there any blockers ? If there is anything I can help with, let me know

**Comment by elopio** [comment]
I am waiting on a review on this to resume testing: https://github.com/keybase/client/pull/7685

**Comment by eli-schwartz** [comment]
Delta updates aside, the current packaging is literally just some standalone distro-agnostic binaries wrapped in a packaging format. As evidenced by the fact that the arch Linux keybase-bin package simply downloads and repackages the deb package. So doing the same repackaging on whichever distro you use should not be hard (though you haven't mentioned which one, so I could be wrong.)

It shouldn't even be hard to wrap *that* in a snap/flatpak wrapper. I mean, this is a bunch of golang binaries alongside an electron app, so there is practically zero system integration re: shared libraries to begin with and not really much point creating another copy of upstream prebuilt binaries created *inside* snap's ecosystem.

**Comment by Mikaela** [comment]
Is the Keybase package at Snapcraft official and still maintained? @elopio doesn't seem to be publicly a member of the Keybase organisation at GitHub (but is a member of Ubuntu, Ubuntu Core and Snapcrafters (of which none has [verified their domain at GitHub](https://help.github.com/en/articles/verifying-your-organizations-domain) though)) and the last update has been almost a year ago.

* https://snapcraft.io/keybase

**Comment by elopio** [comment]
It is not official. I never moved it from the edge channel. I'm not sure if the keybase team is interested in having a snap.

**Comment by funnylookinhat** [comment]
Bumping this again.  Looks like the keybase snap correctly installs the binary, but run_keybase / kbfs is left out.  Is it possible to get that included as well?

**Comment by itsnotsaved** [comment]
Hey @elopio 

See https://github.com/snapcrafters/fork-and-rename-me

In the readme,
> [ ] Make a post in the Snapcraft Forum asking for a transfer of the snap name from you to snapcrafters - link
   [ ] Ask a Snapcrafters admin to fork your repo into github.com/snapcrafters, and configure the repo for automatic publishing into edge on commit

You have said, https://github.com/keybase/client/pull/7685#issuecomment-320848183
> I wanted to add the GUI to the snap before moving it to stable. But that will take some time because I don't know much about electron.

See, those are electron desktop GUI clients.
https://github.com/snapcrafters/wire 
https://github.com/snapcrafters/signal-desktop 
https://github.com/snapcrafters/wordpress-desktop

https://github.com/keybase/client/pull/7685#issuecomment-388089023
>yes. I'm not sure if we want both on the same snap or separate. But back when I was working on this I thought it would be better to first test and release the cli, and then take care of the GUI.

GUI 1st :)

**Comment by anonymouse64** [comment]
Hi folks, I'm working on updating the snap packaging of keybase and am running into some issues getting the GUI running. Specifically it runs and displays a window but just gets stuck on "Loading..." and I can find no logs or anything else which indicate something is wrong. Is this a good issue to discuss this here or should I start a new one to discuss this?

See https://github.com/anonymouse64/keybase-snap/issues/1

Thanks,
Ian
